app/clients/__init__.py

```diff
@@ -53,10 +52,13 @@ def get_email_client(self, name):
         return self.email_clients.get(name)
 
     def get_client_by_name_and_type(self, name, notification_type):
-        assert notification_type in ["email", "sms"]  # nosec B101
+        assert notification_type in {
+            NotificationType.EMAIL,
+            NotificationType.SMS,
+        }  # nosec B101
```

Comment:
The assert used error. Honestly these aren't all that helpful to ignore, since there's thousands of asserts used in the tests. We might want to simply exclude that code in the tests folder...